modules/apache-airflow/main.tf

```diff
@@ -66,7 +66,7 @@ spec:
   sources:
   - repoURL: 'https://airflow.apache.org'
     chart: airflow
-    targetRevision: 1.11.0
+    targetRevision: 1.15.0

```

Comment:
These are not new changes (Git is showing them in the diff for some reason). Please ignore changes here.